app/Plugin/ShibbAuth/Controller/Component/Auth/ApacheShibbAuthenticate.php

```diff
@@ -194,7 +196,11 @@ public function getUserRoleFromGroup($groupTag, $groupRoleMatching, $roleId)
     public function checkOrganization($org, $user)
     {
         $orgModel = ClassRegistry::init('Organisation');
-        $orgAux = $orgModel->find($org);
+        $orgAux = $orgModel->find('first', array(
+                'fields' => array('Organisation.id'),
+                'conditions' => array('name' => $org),
+            )
+        );
```

Comment:
Are you sure that this works as intended? Wouldn't the orgId be in $orgAux['Organisation']['id'] ?